### PR TITLE
feat-added different Input types

### PR DESCRIPTION
**Pull request by prathmeshjagtap**
Added Different Input Types
1) Basic Input
2)Different Size inputs
3)Labeled Inputs
4)Error Success Input
5)Message Input
6)Input With Options
7)Date and time input
8)CheckBox Input
9)Radio Input
10)Text Area
11)sample Form

### CODE REVIEW

**Comment by MiheerTamkhane** [comment]
image



Extra small input code snippet is missing i think?

File: componets/input/input.html [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+            </div>
+            <div class="input__radio">
+              <input id="Green" type="radio" />
+              <label for="Green">Green </label>
+            </div>
+            <div class="input__radio">
+              <input id="yellow" type="radio" disabled />
+              <label for="yellow">yellow </label>
+            </div>
+          </div>
```
Comment (MiheerTamkhane) [inline componets/input/input.html]: Change Id's of radio buttons the checkbox and radio buttons has same id's

File: componets/input/input.html [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+              <div class="input__box">
+                <label for="Password">Password</label>
+                <input id="Password" class="input" />
+              </div>
+              <div class="input__box">
+                <label for="ConfirmPassword">Confirm Password </label>
+                <input id="ConfirmPassword" class="input" />
+              </div>
+              <input type="submit" value="Submit" class="btn btn-primary" />
+            </form>
```
Comment (MiheerTamkhane) [inline componets/input/input.html]: In form inputs add "required" keyword then the form will not submit on empty inputs.

**Review by MiheerTamkhane — APPROVED** [review]
Looks very good man.

**Comment by Mahi24-prog** [comment]
Looks good to me

**Review by Mahi24-prog — APPROVED** [review]

File: componets/input/input.html (hunk 2 above)
Comment (prathmeshjagtap) [reply componets/input/input.html]: yes i have done the changes

File: componets/input/input.html (hunk 1 above)
Comment (prathmeshjagtap) [reply componets/input/input.html]: yes i have fixed the issue

**Comment by prathmeshjagtap** [comment]
> image
> 
> Extra small input code snippet is missing i think?

yes fixed it

**Pull request merged**